src/Core/Components/Button/FluentCompoundButton.razor.cs

```diff
@@ -0,0 +1,162 @@
+    [Parameter]
+    public ButtonSize? Size { get; set; }
+
+    /// <summary>
+    /// Gets or sets the name of the element.
+    /// Allows access by name from the associated form.
+    /// </summary>
+    [Parameter]
+    public string? Name { get; set; }
+
+    /// <summary>
+    /// Gets or sets the visual appearance.
+    /// The default value is `null`. Internally the component uses <see cref="ButtonAppearance.Outline"/> as its default value.
+    /// </summary>
+    [Parameter]
+    public ButtonAppearance? Appearance { get; set; }
+
+    /// <summary>
+    /// Gets or sets the background color of this button (overrides the <see cref="Appearance"/> property).
+    /// Set the value `rgba(0, 0, 0, 0)` to display a transparent button.
+    /// </summary>
+    [Parameter]
+    public string? BackgroundColor { get; set; }
+
+    /// <summary>
+    /// Gets or sets the color of the font (overrides the <see cref="Appearance"/> property).
+    /// </summary>
+    [Parameter]
+    public string? Color { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Icon"/> displayed at the start of button content.
+    /// </summary>
+    [Parameter]
+    public Icon? IconStart { get; set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="Icon"/> displayed at the end of button content.
+    /// </summary>
+    [Parameter]
+    public Icon? IconEnd { get; set; }
+
+    /// <summary>
+    /// Gets or sets the title of the button.
+    /// The text usually displayed in a `tooltip` popup when the mouse is over the button.
+    /// </summary>
+    [Parameter]
+    public string? Title { get; set; }
+
+    /// <summary>
+    /// Gets or sets the element's disabled state, ensuring it doesn't participate in form submission.
+    /// </summary>
+    [Parameter]
+    public bool Disabled { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the value indicating the button is focusable.
+    /// </summary>
+    [Parameter]
+    public bool DisabledFocusable { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets a way to prevent further propagation of the current event in the capturing and bubbling phases.
+    /// </summary>
+    [Parameter]
+    public bool StopPropagation { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets if the button only shows an icon
+    /// Can be used when using <see cref="ChildContent"/> that renders as an icon
+    /// </summary>
+    [Parameter]
+    public bool IconOnly { get; set; }
+
+    /// <summary>
+    ///  Gets or sets the content to be rendered inside the button.
+    ///  This can be used as an alternative to specifying the content as a child component of the button.
+    ///  If both are specified, both will be rendered.
+    ///  </summary>
+    [Parameter]
+    public string? Label { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content to be rendered inside description area of the button.
+    /// </summary>
+    [Parameter, EditorRequired]
```

Comment:
This property doesn't have to be required: the component displays just fine without `Description`, and I think it can be useful in certain cases.